### PR TITLE
Two minor cleanup items

### PR DESCRIPTION
**Pull request by jkotalik**
- Adds auto assign port to the voting sample (was accidentally stashed when I was testing)
- Increase docker stop/remove timeouts and log a warning when we failed to timeout. I frequently hit the 5 second timeout locally, sometimes taking 5 to 10 seconds to stop docker.

### CODE REVIEW

File: samples/voting/tye.yaml [hunk 1]
```diff
@@ -8,6 +8,8 @@ services:
     - port: 6379
 - name: worker
   project: worker/worker.csproj
+  bindings:
```
Comment (davidfowl) [inline samples/voting/tye.yaml]: Undo this change, it's no longer required.

File: src/Microsoft.Tye.Hosting/DockerRunner.cs [hunk 2]
```diff
@@ -207,12 +208,17 @@ async Task RunDockerContainer(IEnumerable<(int Port, int? ContainerPort, int Bin
...
-                var timeoutCts = new CancellationTokenSource(TimeSpan.FromSeconds(5));
+                var timeoutCts = new CancellationTokenSource(DockerStopTimeout);
 
                 _logger.LogInformation("Stopping container {ContainerName} with ID {ContainerId}", replica, shortContainerId);
 
                 result = await ProcessUtil.RunAsync("docker", $"stop {containerId}", throwOnError: false, cancellationToken: timeoutCts.Token);
 
+                if (timeoutCts.IsCancellationRequested)
+                {
+                    _logger.LogWarning($"Failed to stop container after {DockerStopTimeout.Seconds} seconds, container will most likely be running.", replica, shortContainerId);
```
Comment (rynowak) [inline src/Microsoft.Tye.Hosting/DockerRunner.cs]: 👏
